images/linux/toolsets/toolset-1804.json

```diff
@@ -329,5 +329,8 @@
             "name": "yarn",
             "command": "yarn"
         }
-    ]
+    ],
+    "mongodb": {
+        "default": "5"
```

Comment:
maybe version instead of the default? As we don't have multiple versions anyway 😃

Reply:
Sure we can. I just used existing naming =)

Reply:
I should have left the same comment for nodejs, but forgot

Reply:
Updated